docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -142,13 +143,25 @@ $ echo $?
 # 1
 ```
 
-When logging in, the `nginxrestarter` user and any groups that do not already exist
-will be created on the host. The `nginxrestarter` user will be added to the `ubuntu`,
-`nginx` and `other` groups as specified in the `host_groups` field.
-
-A file will also be created in `/etc/sudoers.d` with the contents of the
-`host_sudoers` file written with one entry per line, each prefixed with the username
-of the user that has logged in.
+When the user above logs in, the `nginxrestarter` user and any groups that do
+not already exist will be created on the host. The `nginxrestarter` user will be
+added to the `ubuntu`, `nginx` and `other` groups as specified in the
+`host_groups` field.
+
+The Teleport SSH Service executes `useradd` in order to create new users on the
+host, and returns an error if it cannot find the `useradd` binary. It creates a
+new home directory based on the host user's name and adds the user to the groups
+specified in the Teleport user's roles.
```

Comment:
```suggestion
specified in the user's Teleport roles.
```
minor suggestion.

Reply:
The original makes a stronger distinction that the user we're referring to is the Teleport user, and not the new system user being created.